circle.yml

```diff
@@ -19,11 +19,9 @@ test:
             parallel: true
         
         # Wait for it to have booted
-        - circle-android wait-for-boot
-        
+
         # Tun tests  against the emulator.
-        - ./gradlew connectedCheck
-        
+
```

Comment:
Remove comments, add `./gradlew build`

Remove
```
emulator -avd circleci-android22 -no-audio -no-window:
            background: true
parallel: true
```

Reply:
Remove

```
- mkdir $CIRCLE_TEST_REPORTS/AndroidTests
        - cp -r app/build/reports/androidTests/connected/* $CIRCLE_TEST_REPORTS/AndroidTests
```

And replace this with unit tests report

Reply:
Change this to correct build apk directory `cp -r ${HOME}/${CIRCLE_PROJECT_REPONAME}/app/build/outputs/apk/ $CIRCLE_ARTIFACTS`

Reply:
@iamareebjamal unit tests are not yet initialised so path are you referring to?

Reply:
which path*

Reply:
Unit tests are ran when build command is executed

Reply:
so the reports path will ci path starting with usr/ubuntu/  ?

Reply:
No, just the relative path will change. Please take a look at Orga App config